tools/orxFontGen/build/premake4.lua

```diff
@@ -327,3 +300,22 @@ project "orxFontGen"
             "winmm",
             "OpenGL32"
         }
+

```

Comment:
As only the macosx version is using the dynamic libraries, the copy steps should not be there for windows & linux.